### ISSUE TITLE
Breadcrumbs of rems being searched in search portal are displaced

### ISSUE THREAD

**Issue by anishaaa1** [remnoteio/remnote-issues#59]
**Describe the bug**
With the new updates, portals have been cleaned up but this has brought up a new bug wherein the new mode of breadcrumbs display within a search portal seems to have issues with placement/positioning. Initially, I was unable to locate any breadcrumbs at all, but upon scrolling down, I found the breadcrumbs which were randomly placed somewhere within the search portal.

**To Reproduce**
1. Open a search portal
2. Search for any reference
3. You may notice that breadcrumbs for some rems are not where they're supposed to be (that is, above the respective search result)

**Expected behavior**
Breadcrumbs should appear above the respective search result for all search results as they do for most of the results.

**Screenshots**
![image](https://user-images.githubusercontent.com/68901589/90899761-76471600-e3d9-11ea-983b-48e683d565a7.png)

**Desktop:**
 - OS: MacOS Catalina
 - Browser: Chrome
 - Version: Version 84.0.4147.125 (Official Build) (64-bit)

**Additional context**
N/A

**Comment by mfranzs** [comment]
We've reproduced this and have deployed a fix in the latest update. Please let us know if you're still having any problems.

**Comment by anishaaa1** [comment]
@mfranzs Unfortunately, I'm still having issues. In the below screenshot, I've marked search results and their respective breadcrumbs with the same color. Also, on top, the search result for one of the rems (_~Latin:: Liquor folliculi_) appears above its respective breadcrumbs.
I've tried refreshing the portal and refreshing RemNote as well but to no avail.

![image](https://user-images.githubusercontent.com/68901589/92436561-0517a900-f1b6-11ea-816d-c8e6f1b893f1.png)
